src/transformers/models/hubert/__init__.py

```diff
@@ -21,6 +21,7 @@
 
 
 _import_structure = {
+    ".wav2vec2.feature_extraction_wav2vec2": ["Wav2Vec2FeatureExtractor"],
```

Comment:
This feels all kinds of wrong, @sgugger @LysandreJik what do you think? :sweat_smile: